### DISCUSSION TITLE
indojavstream.com

### DISCUSSION THREAD

**Discussion by Bug-Cache** [realodix/AdBlockID#1176] [Q&A]
**Sample URL**
https://indojavstream.com/watch/jul00781.html?key=wn82z42v5l71

**Penjelasan Lebih Lanjut**

ketika klik play harus mengklik 3x baru bisa, apakah ublock origin support untuk mencegah hal ini
Server ST

![JUL-781__Digoyang_Ipar_Toge_–_Marina_Shiraishi_-__03-05-2022-(379-124)](https://user-images.githubusercontent.com/98657967/166407299-8bebf17f-2447-4cc8-a420-d49cbbfdd646.png)
![image](https://user-images.githubusercontent.com/98657967/166407329-3f0db016-22f0-498d-87e5-6181b7f4f423.png)

**Comment by fgt7** [comment]
saya pakai [fuckfuckadblock](https://github.com/bogachenko/fuckfuckadblock) 1x klik tapi tidak bisa diputar khusus streamtape, selalu loading @bogachenko
